docs/reference/schema-viewer-navigation.md

```diff
@@ -0,0 +1,169 @@
+    readTimeoutMs=200000
+  } # first key-value pair
+
+  ext-airports {
+    type = WebserviceFileDataObject
+    url = "https://davidmegginson.github.io/ourairports-data/airports.csv"
+    followRedirects = true
+    readTimeoutMs=200000
+  } # second key-value pair
+}
+
+...
+
+```
+It is worth noting that the `WebserviceFileDataObject` element appears twice, each time with distinct keys. Initially, it is employed for downloading departure dates, and subsequently for acquiring airport information data.
+
+In contrast to `[mapOf]` elements, elements marked by `[oneOf]` can only accommodate a single nested element. These elements function as a specialized type of property, demanding the inclusion of one specific sub-element.
+
+Example from schema browser (schema{}->actions[mapOf]*->CopyAction{}->executionMode[oneOf]):
+```
+actions {
+  select-airport-cols {
+    type = CopyAction
+    executionMode = { type = DataObjectStateIncrementalMode } # [oneOf] element executionMode containing the nested element of type DataObjectStateIncrementalMode
+    ...
+  }
+}
+
+...
+
+```
+
+
+Arrays or lists use the square brackets notation `[]` in an elements name. They are used for example in the transformer element for creating transformation pipelines that are processed sequentially. 
+
+Example from schema browser (schema{}->actions[mapOf]*->CustomDataFrameAction{}->transformers[]):
+```
+actions {
+  join-departures-airports {
+    type = CustomDataFrameAction
+    inputIds = [int-departures, int-airports]
+    outputIds = [btl-departures-arrivals-airports]
+    transformers = [{
+      type = SQLDfsTransformer
+      code = {
+        btl-connected-airports = """
+          select int_departures.estdepartureairport, int_departures.estarrivalairport,
+            airports.*
+          from int_departures join int_airports airports on int_departures.estArrivalAirport = airports.ident
+        """
+       }
+      },
+      {
+        type = SQLDfsTransformer
+        code = {
+          btl-departures-arrivals-airports = """
+            select btl_connected_airports.estdepartureairport, btl_connected_airports.estarrivalairport,
+              btl_connected_airports.name as arr_name, btl_connected_airports.latitude_deg as arr_latitude_deg, btl_connected_airports.longitude_deg as arr_longitude_deg,
+              airports.name as dep_name, airports.latitude_deg as dep_latitude_deg, airports.longitude_deg as dep_longitude_deg
+            from btl_connected_airports join int_airports airports on btl_connected_airports.estdepartureairport = airports.ident
+          """
+        }
+      }
+    ] # transformer array element containing two SQLDfsTransformer elements
+
+    ...
+
+
+  }
+}
+
+...
+
+```
+
+
+
+Finally, elements ending with `(type)`  denote properties that contain values of a predetermined type, such as string, integer, or other specified data types.
```

Comment:
Form our experience the (type) case is the most confusing case for our users. I would suggest to highlight it and move it upwards. The point is that in contrast to other elements where the param name and possibly its possible values are given, in case of type typically the name of the parent tree element is taken as value of the parameter type. E.g. when specifying a dataObject: it starts with a custom name. The containing block has a type parameter, which value relates to the schema parent node element name. Similar with other element like transformers.

Reply:
The highlighting is done with the :::info  ::: block. 

See as reference https://docusaurus.io/docs/next/markdown-features/admonitions

How much should I move it up?